File: C/Loops/Factorial.c
//User function Template for C

int nFactorial(int n){
    int ans = 1;
    if(n==0){
        return ans;
    }
    while(n>0){
        ans=ans*n;
        n=n-1;
    }
    
    
    //Write your code here
    
    return ans;
}

File: GeeksForGeeks-Problems/JAVASCRIPT/Functional Programming/Factorial.js
// User function Template for javascript

/**
 * @param {number} n
 * @returns {number}
 */

class Solution {
  factorial(n) {
    // code here
    let product = 1;
    while (n > 0) {
      product *= n;
      n -= 1;
    }
    return product;
  }
}
